doc/install/stampede.tex

```diff
@@ -13,32 +13,30 @@
 
 
 
-Setting up ASPECT on TACC Stampede is much easier than on previous systems since
+\noindent Setting up ASPECT on TACC Stampede is much easier than on previous systems since
 most of the necessary libraries are already correctly installed and configured. Following all
 the instructions in this document should allow you to start running ASPECT within 30
-minutes.
+minutes. For further information on Stampede you can also refer to the user guide at \\\url{http://www.tacc.utexas.edu/user-services/user-guides/stampede-user-guide} \\
```

Comment:
// is to make sure the url goes to the next line. Perhaps putting a space between "//" and /url is easier to understand.